test/fixtures/test.go

```diff
@@ -9,4 +9,4 @@ func print(txt string) {
 }
 func main() {
 	print("Hello")
-}
+}
```

Comment:
Removal of this line at the end is breaking one of the tests which tries to add some text there